app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -72,13 +78,18 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * @param menuItems List of [RecentTabMenuItem] shown long clicking a [RecentTab].
  * @param onRecentTabClick Invoked when the user clicks on a recent tab.
  * @param onRecentSearchGroupClicked Invoked when the user clicks on a recent search group.
+ * @param onRecentSyncedTabClick Invoked when the user clicks on the recent synced tab.
+ * @param onSyncedTabSeeAllClick Invoked when user clicks on the "See all" button in the synced tab card.
  */
 @Composable
+@Suppress("LongParameterList")
 fun RecentTabs(
     recentTabs: List<RecentTab>,
     menuItems: List<RecentTabMenuItem>,
     onRecentTabClick: (String) -> Unit = {},
-    onRecentSearchGroupClicked: (String) -> Unit = {}
+    onRecentSearchGroupClicked: (String) -> Unit = {},
+    onRecentSyncedTabClick: (RecentTab.SyncedTab) -> Unit = {},
+    onSyncedTabSeeAllClick: () -> Unit = {},
```

Comment:
```suggestion
    onSyncedTabSeeAllButtonClick: () -> Unit = {},
```
Just wanted to make sure we had noun before the Click.